### PR TITLE
Click event's being unresponsive

### PR DESCRIPTION
**Pull request by jls47**
But this is what I've got so far.

### CODE REVIEW

**Comment by ZenaMel** [comment]
It's a bit hard to follow what's going and what the intent is right now since there are so many changes all across the code. Would you mind writing a couple sentences about what the code ought to do now?

**Comment by jls47** [comment]
Absolutely-probably should have only committed a couple files in the first place.  First, I touched up the background.js to only add an event listener once the user navigates to netflix.com.  Any changes to the manifest were in support of this.  The listener will then execute credits_controller.js when the netflix tab updates to netflix.com/watch/*.  In credits controller, the mutation observer is activated after around 3 seconds to give the browser time to load the actual page and be able to find Netflix's player controls.  In theory, as soon as these controls update to include the watch credits button that button should be clicked; however, I have not been able to successfully fire a click command on the button, nor have I been able to hide it or do much of anything else with it.

**Comment by ZenaMel** [comment]
have you been able to do an alert() once the buttons appear? Because that's something that did work in my original code. I'll have to merge your code on another branch since the original functionality is not being maintained by your changes.

File: migrate-this/credits_controller.js [hunk 1]
```diff
@@ -11,6 +14,46 @@ function notify_button_exists(button) {
...
+
+
+PlayerControlsNeo__all-controls
+PlayerControlsNeo__all-controls
+
+PlayerControlsNeo__layout
+
+*/
+
+setTimeout(function() {
```
Comment (ZenaMel) [inline migrate-this/credits_controller.js:33]: Why would you use a timeout function instead of just wrapping this part of the code inside a try block with a timer?

File: migrate-this/credits_controller.js [hunk 2]
```diff
@@ -11,6 +14,46 @@ function notify_button_exists(button) {
...
+const target = document.getElementsByClassName("PlayerControlsNeo__layout")[0];
```
Comment (ZenaMel) [inline migrate-this/credits_controller.js:34]: It's probably better to use XPaths to get elements from DOM.

File: migrate-this/credits_controller.js [hunk 3]
```diff
@@ -11,6 +14,46 @@ function notify_button_exists(button) {
...
+				document.querySelectorAll('[data-uia="watch-credits-seamless-button"]')[0].click();
+				console.log(document.getElementsByClassName('ltr-qrbw66')[0]);
+				document.getElementsByClassName('ltr-qrbw66')[0].click();
+			}
+        }
+    }
+};
+
+// Create an observer instance linked to the callback function
+const observer = new MutationObserver(callback);
```
Comment (ZenaMel) [inline migrate-this/credits_controller.js:54]: This might be the culprit. Maybe try running your code as a simple endless while loop. No need to optimize for performance atm. KISS.
